### DISCUSSION TITLE
Some properties dont match, Can I manually map these?

### DISCUSSION THREAD

**Discussion by smurfster** [riok/mapperly#1128] [Q&A] [answered]
I have a few classes/domain objects that are use with dapper. Some of the properties on these are not named very well. I have the opportunity to change these within a API Im working on. But the API uses mapperly, I cant seem to find any way of mapping e.g. LstNm -> Surname
In automapper there was a method of mapping these manually, can mapperly do something similar?

**Comment by latonz** [comment, marked as answer]
You can use the `MapProperty` attribute. See the [docs](https://mapperly.riok.app/docs/configuration/mapper/).
